### PR TITLE
feat(farms): Show Boosted APR of address

### PR DESCRIPTION
**Pull request by Woody810**
Should be calculated using the user's boostMultiplier.

before:
image

after:
image

### CODE REVIEW

File: src/views/Farms/components/FarmCard/ApyButton.tsx [hunk 1]
```diff
@@ -72,21 +80,25 @@ const ApyButton: React.FC<React.PropsWithChildren<ApyButtonProps>> = ({
       <Text>
         {t('APR (incl. LP rewards)')}:{' '}
         <Text style={{ display: 'inline-block' }} color={strikethrough && 'secondary'}>
-          {strikethrough ? `${(apr * 2 + lpRewardsApr).toFixed(2)}%` : `${displayApr}%`}
+          {strikethrough ? `${(apr * boostMultiplier + lpRewardsApr).toFixed(2)}%` : `${displayApr}%`}
```
Comment (chefilip) [inline src/views/Farms/components/FarmCard/ApyButton.tsx:83]: Hmm. It's good to show the specific boostMultiplier. However, if the proxy address hasn't been created - user hasn't enabled the proxy farm, I think we don't have a specific boostMultiplier value to show. That's why we show the maximum instead.

Comment (Woody810) [reply src/views/Farms/components/FarmCard/ApyButton.tsx:83]: When the proxy address hasn't been created, or user hasn't enabled the proxy farm, it's still show 2x boostMultiplier, so it won't change.

Comment (chefilip) [reply src/views/Farms/components/FarmCard/ApyButton.tsx:83]: Got it. Thanks for your contribution.

**Review by chefilip — APPROVED** [review]

**Comment by chefilip** [comment]
Hi @Woody810  

Could you help me resolve conflict so I could merge the PR? 

Thanks for your contribution.

**Comment by Woody810** [comment]
> Hi @Woody810
> 
> Could you help me resolve conflict so I could merge the PR?
> 
> Thanks for your contribution.

Done.

**Review by chefilip — APPROVED** [review]

**Pull request merged**